tests/gelf_test.py

```diff
@@ -0,0 +1,123 @@
+#!/usr/bin/env python3
+import os, shutil, time
+import socket, threading
+import zlib, json, glob
+from TestHarness import Node, TestHelper, Utils
+
+###############################################################################################
+# This test starts nodeos process which is configured with GELF logging endpoint as localhost, 
+# receives data from the GELF loggging UDP PORT, and checks if the received log entries match
+# those in stderr log file.
+###########################################################################################
+
+
+GELF_PORT = 24081 
+
+# We need debug level to get more information about nodeos process
+logging="""{
+  "includes": [],
+  "appenders": [{
+      "name": "stderr",
+      "type": "console",
+      "args": {
+        "stream": "std_error",
+        "level_colors": [{
+            "level": "debug",
+            "color": "green"
+          },{
+            "level": "warn",
+            "color": "brown"
+          },{
+            "level": "error",
+            "color": "red"
+          }
+        ],
+         "flush": true
+      },
+      "enabled": true
+    },{
+      "name": "net",
+      "type": "gelf",
+      "args": {
+        "endpoint": "localhost:GELF_PORT",
+        "host": "localhost",
+        "_network": "testnet"
+      },
+      "enabled": true
+    }
+  ],
+  "loggers": [{
+      "name": "default",
+      "level": "debug",
+      "enabled": true,
+      "additivity": false,
+      "appenders": [
+        "stderr",
+        "net"
+      ]
+    }
+  ]
+}"""
+
+logging = logging.replace("GELF_PORT", str(GELF_PORT))
+
+nodeos_run_time_in_sec = 5
+
+node_id = 1
+num_received_logs = 0
+last_received_log = ""
+BUFFER_SIZE = 1024
+
+def gelfServer(stop):
+  s = socket.socket(socket.AF_INET, socket.SOCK_DGRAM)
+  s.settimeout(1)
+  s.bind((TestHelper.LOCAL_HOST, GELF_PORT))
+  while not stop():
+    try:
+        data, clienaddr = s.recvfrom(BUFFER_SIZE)
```

Comment:
Since `clienaddr` isn't used, name it `_`.

Reply:
done